packages/admin/admin-theme/src/componentsTheme/MuiLinearProgress.ts

```diff
@@ -0,0 +1,12 @@
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiLinearProgress: GetMuiComponentTheme<"MuiLinearProgress"> = (component) => ({
+    ...component,
+    defaultProps: {
+        color: "primary",
+        sx: {
+            height: "2px",
+            zIndex: 1, // for use in LoadingOverlay of DataGrid, otherwise the LinearProgress is behind the border
+        },
```

Comment:
Please use the `styleOverrides` for styling, instead of passing `sx` through the default props.  
Otherwise the `sx` prop will be replaced when adding additional styling with the `sx` prop, instead of merging the styles.